docs/netlify-dev.md

```diff
@@ -68,22 +68,21 @@ USAGE
   $ netlify dev
 
 OPTIONS
-  -c, --command=command      command to run
-  -f, --functions=functions  Specify a functions folder to serve
-  -o, --offline              disables any features that require network access
-  -p, --port=port            Specify port of netlify dev
-  -l, --live                 Start a public live session
+  -c, --command=command                                        command to run
```

Comment:
Aligned this with the current output of `ntl dev --help` and removed less important flags